res/openvpn-gui-res-it.rc

```diff
@@ -131,9 +131,9 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_ITALIAN, SUBLANG_DEFAULT
 BEGIN
     GROUPBOX "   ", 201, 6, 46, 235, 52
-    AUTORADIOBUTTON "Usa il file di &configurazione di OpenVPN", ID_RB_PROXY_OPENVPN,
+    AUTORADIOBUTTON "Usa file &configurazione OpenVPN", ID_RB_PROXY_OPENVPN,
```

Comment:
I still believe here there is a "di" missing. Shouldn't this be "file di configurazione"? https://it.wikipedia.org/wiki/File_di_configurazione

Reply:
For me should be better 

  AUTORADIOBUTTON "Usa il file di &configurazione di OpenVPN", ID_RB_PROXY_OPENVPN,

But to close this issue I will change it.